src/styles/stars.scss

```diff
@@ -0,0 +1,25 @@
+.card__info-stars {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.stars {
+  display: flex;
+  &__star {
+    background-image: url(../images/star.svg);
+    background-position: center;
+    background-repeat: no-repeat;
+    height: 16px;
+    width: 16px;
+  }
+  &__star:not(:last-child) {
+    margin-right: 4px;
+  }
+
+  &--4 &__star:nth-child(-n+4) {
+  background-image: url(../images/star-active.svg);
```

Comment:
What about other modifiers? It should work correctly with them too
![image](https://github.com/mate-academy/layout_product-cards/assets/96409563/c091874c-012c-4b07-b8d0-850968fe157d)